src/main/java/uk/ac/ebi/eva/pipeline/io/mappers/AccessionReportLineMapper.java

```diff
@@ -22,7 +22,7 @@
 import java.util.HashSet;
 import java.util.Set;
 
-public class AccessionReportLineMapper implements LineMapper<Variant> {
+public class AccessionReportLineMapper extends VariantVcfFactory implements LineMapper<Variant> {
```

Comment:
I am still not fully sold on the idea of re-creating the variant from the accessioning report as opposed to reading SS IDs from the database. Could you test if the ambiguity reported here: https://www.ebi.ac.uk/panda/jira/browse/EVA-1579 when generating the accessioning report is properly dealt with?

Reply:
updated to take care of the case mentioned